randobot/handler.py

```diff
@@ -207,7 +207,7 @@ async def ex_rollseed(self, args, message):
         self.state["permalink_available"] = True
 
         await self.send_message(f"Permalink: {permalink}")
-        await self.set_raceinfo(f" - {permalink}")
+        await self.set_raceinfo(f" - {permalink}", False, False)
```

Comment:
`set_raceinfo` already puts in a `|`: https://github.com/wooferzfg/racetime-bot/blob/faa91296d6703c1cc23db295a16305ee07d7e3b6/racetime_bot/handler.py#L169

```suggestion
        await self.set_raceinfo(permalink, False, False)
```